django_socio_grpc/tests/grpc_test_utils/fake_grpc.py

```diff
@@ -99,16 +99,6 @@ async def read(self):
         except queue.Empty:
             return grpc.aio.EOF
 
-    def __aiter__(self):
```

Comment:
why deleting this ? This is still valid for response streaming no ?

Reply:
I implemented the other calls where each has its own good specs